ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc

```diff
@@ -388,13 +397,65 @@ TorchCore::processIValue (torch::jit::IValue value, GstTensorMemory *output, uns
   output_tensor = output_tensor.contiguous ();
 
   /* validate output tensor once */
-  if (configured < outputTensorMeta.num_tensors && validateOutputTensor (output_tensor, idx)) {
+  if (configured < outputTensorMeta.num_tensors
+&& validateOutputTensor (output_tensor, idx)) {
```

Comment:
Strange indentation...

Reply:
I am not sure why this clang produces this - fixed manually.